fastify.d.ts

```diff
@@ -146,6 +146,7 @@ declare namespace fastify {
     id: any
 
     ip: string
+    ips: any
```

Comment:
Can this really be a `any`? What's the output of the `proxyAddr.all` function?

Reply:
proxyAddr.all returns an array strings. Updated the type to reflect that. efef8cb